exdb/static/exdb/css/ra-home.css

```diff
@@ -0,0 +1,10 @@
+.experience-list , .experience-list td,
```

Comment:
You should put each of these selectors on separate lines. That is easier to read and edit.